EFI.md

```diff
@@ -16,13 +16,13 @@ T2 systems boot EFI like their predicessors.  Unlike previous versions they use
 
 ## Apple Silicon
 
-Apple Silicon don't run EFI but iBoot instead.  (Though it does emulate parts of the EFI system via IOKit for back compatibility)
-However, you can chainload to another bootloader if you want to, it's not prevented when Secure Boot is set to permissive.
-That permissive option doesn't technically turn off Secure Boot. It allows you to enroll your own hashes to the Secure Boot policy on the system w/ user authentication instead.
+Apple Silicon chips don't run EFI but run iBoot instead (Though it does emulate parts of the EFI system via IOKit for back compatibility).
+Even though it's iboot, you can still chainload to another bootloader if you want to; It's not prevented when Secure Boot is set to permissive.
+That permissive option doesn't technically turn off Secure Boot, but allows you to enroll your own hashes to the Secure Boot policy on the system w/ user authentication instead.
```

Comment:
Confusing "permissive" with "fuOS".  Permissive puts XNU in a legacy intel like security state.  fuOS or `coih` is the chain loading mechanism

Reply:
`coih` / `fuOS` - chainloading
`smb0` - `smb3` - Secure Multi Boot - iOS like immutable kernel, these are switched on for "permissive" to have MDM and kext loading
`sip0` - `sip3` - classical SIP settings

all verified by iBoot before execution is passed to XNU